### PR TITLE
togglebar redesign for further changes

### LINKED ISSUE #792: [CORE] Implement updated filter bar design
NOTE: All development and PRs for this task should be done against the branch `feature/core-header-refresh`.

The goal of this task is to modify `ToggleBar.js` to implement the new designs for the filter selectors as shown below.

Mocks: https://www.figma.com/file/H08AQJSaAIRxfeMP4iCUPh/UP-MVP?node-id=1521%3A43

This project is broken up into three sub-tasks for scoping:
- https://github.com/Recidiviz/pulse-dashboard/issues/794
- https://github.com/Recidiviz/pulse-dashboard/issues/796
- https://github.com/Recidiviz/pulse-dashboard/issues/797

### PR DESCRIPTION
**Pull request by AlexK-ino**
## Description of the change

changed tooglebar.js styles for the new design

## Type of change

- [X] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Closes #792 

### Development

These boxes should be checked by the submitter prior to merging:

- [X] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [ ] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: shared-filters/yarn.lock [hunk 1]
```diff
@@ -1062,6 +1062,11 @@ cssstyle@^2.2.0:
   dependencies:
     cssom "~0.3.6"
 
+d3-array@2.8.0:
+  version "2.8.0"
+  resolved "https://registry.yarnpkg.com/d3-array/-/d3-array-2.8.0.tgz#f76e10ad47f1f4f75f33db5fc322eb9ffde5ef23"
+  integrity sha512-6V272gsOeg7+9pTW1jSYOR1QE37g95I3my1hBmY+vOUNHRrk9yt4OTz/gK7PMkVAVDrYYq4mq3grTiZ8iJdNIw==
+
```
Comment (Catacola) [inline shared-filters/yarn.lock:1069]: It's not clear where these additional packages came from.

File: src/components/toggles/ToggleBar.js [hunk 2]
```diff
@@ -41,6 +41,7 @@ import "./ToggleBar.scss";
 const TOGGLE_STYLE = {
   zIndex: 700,
   top: 65,
+  backgroundColor: "#F4F5F5",
```
Comment (Catacola) [inline src/components/toggles/ToggleBar.js:44]: You can use `$core-background` in `src/assets/styles/spec/settings/baseColors.scss` for this value.

**Review by Catacola — CHANGES_REQUESTED** [review]
Hi Alex! Thanks for getting started on this.

There was a mistake on my end. The task linked to this was supposed to be a master task for 3 smaller tasks describing the individual changes needed: 
- #794
- #796
- #797

These should give a much better description of the changes required.

File: src/components/toggles/ToggleBar.js (hunk 2 above)
Comment (AlexK-ino) [reply src/components/toggles/ToggleBar.js]: Thank you!
I didn't pay attention to the variables

File: shared-filters/yarn.lock (hunk 1 above)
Comment (AlexK-ino) [reply shared-filters/yarn.lock]: Oops, sorry - those are redundant modules, I got rid of them

**Comment by AlexK-ino** [comment]
@Catacola Yes, I get it now. That's what @MeloncholyShiro  is doing now, and when he's done, we'll merge it.

**Comment by Catacola** [comment]
@AlexK-ino Thanks for your additional work on this. I pulled much of this together in some separate work and I'll merge yours in there. I'll take the rest of this issue from here. Thanks for bringing it to this point.

**Pull request closed without merging**